### PR TITLE
Adding Nginx load-balancer documentation.

### PR DESCRIPTION
**Pull request by sheyd**
### What's this PR do?

This pull request adds documentation for our Nginx load-balancer (which handles root/misc domain redirects and TLS certificates for said domains).

### How should this be reviewed?

Already reviewed by @DFurnes [here](https://www.pivotaltracker.com/story/show/172442105). Give it a read, already tested steps in production.

### Relevant tickets

References [Pivotal #172442105](https://www.pivotaltracker.com/story/show/172442105).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.

### CODE REVIEW

File: docs/LOAD-BALANCER.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Documentation for Load-Balancer:
+
+## Server Setup Instructions:
+* Setup [Nginx setup guide)(https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04) for Ubuntu 18.04 on a new EC2 instance. *Supplementary step*: Add [Nginx](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/) PPA to run latest stable Nginx version, instead of the default OS Nginx, since it's not updated as frequently and can lag on some feature/security fixes.
```
Comment (weerd) [inline docs/LOAD-BALANCER.md:4]: Your markdown link is borked. Replace closing `)` with `]`.

Comment (weerd) [reply docs/LOAD-BALANCER.md:4]: Also just reads weird as "Setup Nginx setup guide..." Maybe just "Setup Nginx..."?

File: docs/LOAD-BALANCER.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* [Setup Nginx](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04) for Ubuntu 18.04 on a new EC2 instance. *Supplementary step*: Add [Nginx](https://www.nginx.com/resources/wiki/start/topics/tutorials/install/) PPA to run latest stable Nginx version, instead of the default OS Nginx, since it's not updated as frequently and can lag on some feature/security fixes.
+* Setup LetsEncrypt according to [this](https://www.digitalocean.com/community/tutorials/how-to-secure-nginx-with-let-s-encrypt-on-ubuntu-18-04) guide on the server.
+* The Elastic IP assigned to the load-balancer instance _must_ be: `54.172.90.245`.
+
+## Server Administration:
+* How to login to the Nginx Load-balancer server: 
+	* Login to bastion server: `ssh aws-admin` (if you have the SSH config setup), or `ssh ec2-user@jump.dosomething.org`.
+	* `ssh nginx-lb`. In case the `/etc/hosts` file is borked, you can also use `ssh 10.100.60.26`. 
+* All Nginx server config files live in `/etc/nginx/conf.d` and take the filname format: `example-com.conf`.
+* To check Nginx config: `sudo nginx -T`.
```
Comment (DFurnes) [inline docs/LOAD-BALANCER.md:13]: `filname` should be `filename`

File: docs/LOAD-BALANCER.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## How to setup a brand new domain for redirect:
+* Copy-pasta the following entry, and replace `example.com`:
+```
+server {
+    server_name example.com; # This line can contain multiple domain entries, e.g. example.com www.example.com
+    return 302 https://www.newdomain.com; # If you want to pass on the URI, you can use the format: https://www.newdomain$request_uri
+}
+```
+* Generate the server certificate using: `sudo certbot --nginx -d example.com`. You can pass additional `-d example2.com` flags to the cerbot command to generate a certificate per domain.
```
Comment (DFurnes) [inline docs/LOAD-BALANCER.md:24]: `cerbot` should be `certbot`

File: docs/LOAD-BALANCER.md (hunk 2 above)
Comment (sheyd) [reply docs/LOAD-BALANCER.md:13]: Fix is [here](https://github.com/DoSomething/infrastructure/pull/240).

File: docs/LOAD-BALANCER.md (hunk 3 above)
Comment (sheyd) [reply docs/LOAD-BALANCER.md:24]: Gah, sorry @DFurnes! I accidentally re-introduced this after fixing it locally for some reason. 🤦 Fix is [here](https://github.com/DoSomething/infrastructure/pull/240).

**Pull request merged**